pkg/webhook/v1alpha3/pod/inject_webhook.go

```diff
@@ -244,7 +244,7 @@ func wrapWorkerContainer(pod *v1.Pod, jobKind, metricsFile string, mc common.Met
 		if c.Args != nil {
 			args = append(args, c.Args...)
 		}
-		redirectStr := fmt.Sprintf(" 2>&1 | tee %s", metricsFile)
+		redirectStr := fmt.Sprintf("1>%s 2>&1", metricsFile)
```

Comment:
How do we let users know that `kubectl logs` works on the side car which outputs logs from worker container too?  Should we rename the side car container to "logger" or something?
WDYT? @gaocegege @hougangliu

Reply:
SGTM

Reply:
SGTM